src/components/Switch/Switch.stories.tsx

```diff
@@ -26,35 +32,50 @@ export default {
                 type: 'text',
             },
         },
+        mode: {
+            options: SwitchModeTypes,
+            control: { type: 'radio' },
+        },
+        onChange: {
+            table: { disable: false },
+            action: 'Change',
+        },
     },
     args: {
-        on: true,
+        mode: 'on',
```

Comment:
Yes this is a known issue, it's the same in Checkbox story. @fabriciomendonca do you think it's worth trying to fix this?

Reply:
I found a solution for this and believed it was introducing another issue on Storybook: code snippets showing the story code instead of the component usage in some cases. It happens in some components and in the first render of the page, refreshing it solves the issue. 

I will bring the topic and the solution for the controls and code snippets sync into discussion internally and if we decide to use it, I will recreate the commit with the fix: adding `@storybook/preview-api@7.0.0-beta.53` and using the `useArgs` hook instead of `useState` and `useEffect` to update the `mode` value. [Check this commit](https://github.com/Frontify/fondue/pull/1317/commits/57390d76db9649dcc8510b5bb0d3199dfbc8013e#diff-5630dc06a984553378432b84eaba3c19ee3b30756088044395a5a497e83fa07e)

Reply:
Marking this as resolved as this issue will be tackled separately.